infresnel_travel_time fails when loading stored NetCDF due to coordinate order mismatch

When using infresnel_travel_time with a previously stored NetCDF result, the function can raise:

AssertionError: Coordinate names differ!

even though the coordinates themselves are identical. This caused by comparing xarray coordinates by iteration order, which is not may not be preserved after saving and reloading a netcdf file. Maybe this is caused by an update to how xarray works. I have xarray version 2025.12.0 installed.

The bug occurs in lines 377-383 of travel_time.py - 


```
# Various checks to ensure that what we're loading in makes sense
assert diff_path_lens.shape == diff_path_lens_loaded.shape, 'Shapes differ!'
for coord_da, coord_loaded_da in zip(
    diff_path_lens.coords.values(), diff_path_lens_loaded.coords.values()
):
    assert coord_da.name == coord_loaded_da.name, 'Coordinate names differ!'
    assert (coord_da.data == coord_loaded_da.data).all(), f'Coordinate data differ for {coord_da.name}!'
```


Printing shows that coord_da iterates in the order: station, y, x
While coord_da_loaded iterates in the order: y, x, station
So there is a mismatch and an error is raised.

Replacing the above code block with the following fixes:

```
assert set(diff_path_lens.coords) == set(diff_path_lens_loaded.coords), 'Coordinate names differ!'
for name in diff_path_lens.coords:
    assert (diff_path_lens.coords[name].data == diff_path_lens_loaded.coords[name].data).all(), f'Coordinate data differ for {name}!'
```

--- a/rtm/travel_time.py
+++ b/rtm/travel_time.py
@@ -376,11 +376,9 @@ def infresnel_travel_time(grid, st, celerity=343, stored_result=None, dem_file=N
             diff_path_lens_loaded = diff_path_lens_loaded.sel(station=[tr.id for tr in st])
             # Various checks to ensure that what we're loading in makes sense
             assert diff_path_lens.shape == diff_path_lens_loaded.shape, 'Shapes differ!'
-            for coord_da, coord_loaded_da in zip(
-                diff_path_lens.coords.values(), diff_path_lens_loaded.coords.values()
-            ):
-                assert coord_da.name == coord_loaded_da.name, 'Coordinate names differ!'
-                assert (coord_da.data == coord_loaded_da.data).all(), f'Coordinate data differ for {coord_da.name}!'
+            assert set(diff_path_lens.coords) == set(diff_path_lens_loaded.coords), 'Coordinate names differ!'
+            for name in diff_path_lens.coords:
+                assert (diff_path_lens.coords[name].data == diff_path_lens_loaded.coords[name].data).all(), f'Coordinate data differ for {name}!'
             assert diff_path_lens.attrs == diff_path_lens_loaded.attrs, 'Attributes differ!'
             assert diff_path_lens.dem_file == diff_path_lens_loaded.dem_file, 'DEM files differ!'
             print('----------------------------------------------------------------')

